tools/backend/new-lsq-generator/configs.py

```diff
@@ -0,0 +1,93 @@
+import math
+import json
+import sys
```

Comment:
@lucas-rami do we add a license header also to python files?

Reply:
I guess we should yeah, all source files generally-speaking. I don't think any of our RTL files have one but technically they should.